cfgov/jinja2/v1/_includes/organisms/info-unit-group-2.html

```diff
@@ -59,19 +64,28 @@
     {{ value.intro | safe }}
 {%- endif %}
 
-    <div class="content-l">
-    {% for unit in value.info_units %}
+{% for unit in value.info_units %}
+    {% if ( loop.index == 1 ) or
+          ( value.format == '50-50' and loop.index % 2 == 1 ) or
+          ( value.format == '33-33-33' and loop.index % 3 == 1 ) 
+    %}
+    <div class="content-l
+                {{' content-l__top-divider' if value.lines_between_items else ''}}">
```

Comment:
Suggestion, using `{{-` here would eliminate the extra whitespace in the rendered html.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3476/commits/4c649330961585504690e2ccfe018ae41b4e57cc -- thanks!